oximeter/collector/src/lib.rs

```diff
@@ -11,14 +11,19 @@ use dropshot::{
     HttpResponseUpdatedNoContent, HttpServer, HttpServerStarter,
     RequestContext, TypedBody,
 };
+use internal_dns_client::{
+    multiclient::{ResolveError, Resolver},
+    names::{ServiceName, SRV},
+};
```

Comment:
FYI, these utilities show up in https://github.com/oxidecomputer/omicron/pull/1177